x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/analytics_list/action_clone.test.ts

```diff
@@ -0,0 +1,254 @@
+        description: "Outlier detection job with 'glass' dataset",
+        source: {
+          index: ['glass_withoutdupl_norm'],
+          query: {
+            match_all: {},
+          },
+        },
+        dest: {
+          index: 'dest_glass_1',
+          results_field: 'ml',
+        },
+        analysis: {
+          outlier_detection: {
+            compute_feature_influence: true,
+            outlier_fraction: 0.05,
+            standardization_enabled: true,
+          },
+        },
+        analyzed_fields: {
+          includes: [],
+          excludes: ['id', 'outlier'],
+        },
+        model_memory_limit: '1mb',
+        allow_lazy_start: false,
+      };
+      expect(isAdvancedConfig(formCreatedOutlierDetectionJob)).toBe(false);
+    });
+
+    test('should detect a regression job created with the form', () => {
+      const formCreatedRegressionJob = {
+        description: "Regression job with 'electrical-grid-stability' dataset",
+        source: {
+          index: ['electrical-grid-stability'],
+          query: {
+            match_all: {},
+          },
+        },
+        dest: {
+          index: 'dest_grid_1',
+          results_field: 'ml',
+        },
+        analysis: {
+          regression: {
+            dependent_variable: 'stab',
+            prediction_field_name: 'stab_prediction',
+            training_percent: 20,
+            randomize_seed: -2228827740028660200,
+          },
+        },
+        analyzed_fields: {
+          includes: [],
+          excludes: [],
+        },
+        model_memory_limit: '150mb',
+        allow_lazy_start: false,
+      };
+
+      expect(isAdvancedConfig(formCreatedRegressionJob)).toBe(false);
+    });
+
+    test('should detect advanced classification job', () => {
+      const advancedClassificationJob = {
+        description: "Classification job with 'bank-marketing' dataset",
+        source: {
+          index: ['bank-marketing'],
+          query: {
+            match_all: {},
+          },
+        },
+        dest: {
+          index: 'dest_bank_1',
+          results_field: 'CUSTOM_RESULT_FIELD',
+        },
+        analysis: {
+          classification: {
+            dependent_variable: 'y',
+            num_top_classes: 2,
+            prediction_field_name: 'y_prediction',
+            training_percent: 2,
+            randomize_seed: 6233212276062807000,
+          },
+        },
+        analyzed_fields: {
+          includes: [],
+          excludes: [],
+        },
+        model_memory_limit: '350mb',
+        allow_lazy_start: false,
+      };
+
+      expect(isAdvancedConfig(advancedClassificationJob)).toBe(true);
+    });
+
+    test('should detect advanced outlier_detection job', () => {
+      const advancedOutlierDetectionJob = {
+        description: "Outlier detection job with 'glass' dataset",
+        source: {
+          index: ['glass_withoutdupl_norm'],
+          query: {
+            // TODO check default for `match`
```

Comment:
Did you mean to leave in this comment?

Reply:
I noticed one of the outlier detection jobs (created by env bootstrap) has query `match` instead of `match_all`. As I saw in all the other cases `match_all` is a default, so probably it's just a typo in a bootstrap job.